abc.templates/deployments/contents/deploy.sh

```diff
@@ -0,0 +1,10 @@
+IMAGE_NAME=REPLACE_FULL_IMAGE_NAME:REPLACE_VERSION
```

Comment:
This needs to be executable and a script.

Add `#!/usr/bin/env bash`
Add `set -euo pipefail`